app/src/test/java/org/simple/clinic/summary/PatientSummaryUpdateTest.kt

```diff
@@ -230,14 +230,14 @@ class PatientSummaryUpdateTest {
   }
 
   @Test
-  fun `when there are patient summary changes and no measurements are recorded and warning is shown, clicking on back link id with patient screen must go back to home screen`() {
+  fun `when there are no patient measurement changes and warnings, clicking on back link id with patient screen must go back to home screen`() {
```

Comment:
In all the test cases we are going back to the previous screen or the home screen. Don't you think we should also write a test case for when the appointment sheet is shown?

Reply:
If there are no changes we don't show the appointment sheet, but we show if there are changes. We have a test for that case though.